### ISSUE TITLE
CodeRabbit AI PR reviews

### ISSUE THREAD

**Issue by tvolk131** [fedimint/fedimint#4775]
[CodeRabbit](https://coderabbit.ai/) offers free PR reviews for public GitHub repos. I've used it on a small project before and it seems helpful. I'm not sure how it would handle a larger codebase but I'm curious if people are interested in trying this out.

**Comment by elsirion** [comment]
Definitely! I was playing with manually dumping change sets into GPT4 when it came out. Wasn't worth it, but this might be since it reduces the friction!

**Comment by elsirion** [comment]
I activated it, let's see how it goes.

**Comment by justinmoon** [comment]
dev call: we deactivated because it was noisy. code rabbit may be more configurable and we could silence it a little.

**Comment by elsirion** [comment]
Removed, everyone got annoyed by it.

**Comment by tvolk131** [comment]
Meh, GitHub Copilot can review PRs now anyway
